### DISCUSSION TITLE
Why was there no release for December?

### DISCUSSION THREAD

**Discussion by ArlindH** [python-gitlab/python-gitlab#2447] [Q&A] [answered]
Hello,

I was trying to understand why [resource groups](https://python-gitlab.readthedocs.io/en/main/gl_objects/resource_groups.html) are in the docs but could not get it to work with the installed Package.

Came across #2398 which was merged on December 18th. According to the [contributing docs](https://github.com/python-gitlab/python-gitlab/blob/main/CONTRIBUTING.rst), there should have been a release on December 28th, because the commit message contains `feat` in the name, thus signaling a semantic version bump. 

Is there something that I can do to trigger a release so that release groups can be made available?

Thank you

**Comment by nejch** [comment, marked as answer]
Sorry about that @ArlindH, I paused the release workflow during the holidays as we were all away and I wanted to get some ongoing work related to HTTP backends merged in here before the release. But I can re-enable and run the release workflow in a bit.

**Reply by ArlindH** [reply]
No worries @nejch!

Will be waiting for the release. 

Thanks a lot for all your work!

**Reply by nejch** [reply]
@ArlindH took a while this time, but we had a release at the end of the month :)
